Fix build failure on Mac OS X Lion 10.7:

In file included from src/globalsosx.cpp:16:
/usr/include/mach/task_info.h:252:2: error: unknown type name 'vm_extmod_statistics_data_t'
        vm_extmod_statistics_data_t             extmod_statistics;
        ^
1 error generated.
--- GDE/MUSCLE/src/globalsosx.cpp.orig	2016-05-02 06:19:05.000000000 -0500
+++ GDE/MUSCLE/src/globalsosx.cpp	2020-10-10 06:41:58.000000000 -0500
@@ -13,10 +13,10 @@
 #include <netinet/icmp6.h>
 #include <sys/vmmeter.h>
 #include <sys/proc.h>
+#include <mach/vm_statistics.h>
 #include <mach/task_info.h>
 #include <mach/task.h>
 #include <mach/mach_init.h>
-#include <mach/vm_statistics.h>
 
 const double DEFAULT_RAM = 1e9;
 const double DEFAULT_MEM_USE = 1e6;
